app/konnect/updates.md

```diff
@@ -32,6 +32,10 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 * decK command flag: `--konnect-runtime-group-name` to `--konnect-control-plane-name`
 * decK state file attribute: `_konnect.runtime_group_name`  to` _konnect.control_plane_name`
 
+**Gateway Manager redesigns**
+: The Gateway Manager Landing Page is now updated to nudge customers to create a control plane if none are present. In addition, the control plane creation workflow has been revamped to make it more intuitive and easier to follow.
+
+: The Gateway Control Plane Overview page is now updated to present the users all relevant information of a specific control plane. In addition, prompts that nudge users to create their first service, route, plugin and consumer are presented on the Overview Pages to nudge users to immediately dive into {{site.konnect_short_name}} for the first time.
```

Comment:
```suggestion
: The Gateway Control Plane Overview page is now updated to present the users all relevant information of a specific control plane. In addition, prompts that nudge users to create their first service, route, plugin, and consumer are presented on the Overview Pages to nudge users to immediately dive into Konnect for the first time.
```
Shouldn't use variables in release notes, since they're meant to be a snapshot in time.